spec/rubocop/cop/rspec/let_before_examples_spec.rb

```diff
@@ -41,10 +41,23 @@
     RUBY
   end
 
-  it 'flags `let` after `include_examples`' do
+  it 'ignores `let` after `include_examples`' do
+    # NOTE: include_examples may define the same variable as `let`,
+    # and changing the order may break the spec due to order dependency
+    # if `let` is moved above.
+    expect_no_offenses(<<-RUBY)
```

Comment:
Looking at the condition `if if multiline_block?(node.body)` in the original code, I have the feeling that this may fail

```
RSpec.describe User do
         include_examples('should be BEFORE let as it defines `let(:foo)`, too')
       end
     RUBY
```

Reply:
I couldn't make it fail. As I've moved the check below `multiline_block?`, there should be no issues.